### PR TITLE
Add 3.2 release announcement

### PR DESCRIPTION
**Pull request by johnpmitsch**
announcing 3.2 and fixing some grammar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -65,10 +65,13 @@ <h1 class="tagline col-md-5">
       </div>
       <div class="col-md-3 col-md-offset-1 news">
         <p class="snippet">
-        Katello 3.1.0 is has been officially released to address upgrade issues! Please <a href="/docs/3.1/release_notes/release_notes.html">read the release notes</a> or <a href="/docs/3.1/upgrade/index.html">upgrade instructions.</a> before getting started.
+        Katello 3.2.0 is officially released! Please  <a href="/docs/3.2/release_notes/release_notes.html">read the release notes</a> or <a href="/docs/3.2/upgrade/index.html">upgrade instructions.</a> before getting started.
```
Comment (cfouant) [inline index.html]: There's an extraneous period in this line after 'upgrade instructions'

**Comment by johnpmitsch** [comment]
@cfouant good catch! Fixed in all announcements

**Comment by cfouant** [comment]
@johnpmitsch - it seems like all the links are broken. There is no 3.2 directory inside /docs. Is this a future feature, or do the links need fixing?

**Comment by johnpmitsch** [comment]
> @johnpmitsch - it seems like all the links are broken. There is no 3.2 directory inside /docs. Is this a future feature, or do the links need fixing?

@cfouant  I believe that has to do with the way deployment works for katello.org, @ehelms can confirm for sure. If that is the case, you can test out the links on the actual katello.org and just copy and paste them in.

**Comment by ehelms** [comment]
Correct, the way it deploys today it will build everything properly for us. And keep in mind most of this is mid-move to theforeman.org anyway, I'm just waiting on someone to review my PRs to finalize it.

**Comment by cfouant** [comment]
Sounds good. ACK from me. Not sure if you need someone with write capabilities to ACK as well.

**Pull request merged**